docs/reference/remap/to_string.cue

```diff
@@ -0,0 +1,49 @@
+package metadata
+
+remap: functions: to_string: {
+	arguments: [
+		{
+			name:        "value"
+			description: "The value to return a string representation of."
+			required:    true
+			type: ["any"]
+		},
+		{
+			name:        "default"
+			description: "If the value parameter errors, return this parameter instead."
+			required:    false
+			type: ["any"]
+		},
+	]
+	return: ["boolean", "integer", "float", "string", "map", "array", "null"]
```

Comment:
This could be a problem with the function and not the docs, but it seems a bit strange that `to_string` would return any type. If the first parameter doesn't coerce to a string, the second parameter is returned unprocessed - so it can be any type..

Reply:
The function is "correct", in that it can covert any value to a string (regardless of how useful that is).

The `default` is still useful if you provide a path that doesn't resolve to anything.

Still though, we might limit `value` to anything _except_ maps and arrays.